sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/ConnectionStringParser.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Messaging.ServiceBus.Core
+{
+    /// <summary>
+    ///   Allows for parsing Service Bus connection strings.
+    /// </summary>
+    ///
+    internal static class ConnectionStringParser
+    {
+        /// <summary>The character used to separate a token and its value in the connection string.</summary>
+        private const char TokenValueSeparator = '=';
+
+        /// <summary>The character used to mark the beginning of a new token/value pair in the connection string.</summary>
+        private const char TokenValuePairDelimiter = ';';
+
+        /// <summary>The formatted protocol used by an Service Bus endpoint. </summary>
+        private const string EventHubsEndpointScheme = "sb://";
```

Comment:
nit:  Should rename the members with `EventHubs` in them.

Reply:
Updated